os/net/bluetooth/bluetooth.c

```diff
@@ -328,8 +312,11 @@ static int bt_init(void)
 		return ret;
 	}
 
-	bt_finalize_init();
+#ifdef CONFIG_BLUETOOTH_NULL
+	write_local_name("TizenRT");
+#endif
 
+	bt_finalize_init();
```

Comment:
'finalize' is not good naming, currently bt_finalize_init make bt as a ready state, then bt_set_ready or something will be better.

Reply:
This function is from Zephyr's opensource, and we are porting Zephyr's code into TizenRT
https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/host/hci_core.c